hudi-spark-datasource/hudi-spark3/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/Spark32HoodieParquetFileFormat.scala

```diff
@@ -225,6 +239,10 @@ class Spark32HoodieParquetFileFormat(private val shouldAppendPartitionValues: Bo
       if (enableVectorizedReader) {
         val vectorizedReader =
           if (shouldUseInternalSchema) {
+            val int96RebaseSpec =
+              DataSourceUtils.int96RebaseSpec(footerFileMetaData.getKeyValueMetaData.get, int96RebaseModeInRead)
+            val datetimeRebaseSpec =
+              DataSourceUtils.datetimeRebaseSpec(footerFileMetaData.getKeyValueMetaData.get, datetimeRebaseModeInRead)
```

Comment:
this implies schema evolution only works with 3.2.1 ? cc @xiaorixiaoyao

Reply:
yes，
current schema evoluton work on  spark3.1.x  and   >=  spark3.2.1 
for the subsequent evolution。
from spark3.2.x,   schema evolution choose to rely on dsv2.  since dsv2 only work with spark version >= 3.2.1,  so schema evolution cannot support spark3.2.0